paddle/fluid/framework/operator.cc

```diff
@@ -1007,6 +1007,15 @@ static void CheckTensorNANOrInf(const std::string& op_type,
                               op_type, name));
 }
 
+bool OperatorWithKernel::SupportMKLDNN() const {
```

Comment:
I know there is already the method named `SupportGPU()`, but I would prefer naming `SupportsMKLDNN()` (indicative) instead of `SupportMKLDNN()` (imperative).

Reply:
Also, maybe we could hide it inside the `CanMKLDNNBeUsed()` method?

Reply:
CanMKLDNNBeUsed() function uses Runtime context at layer level and SupportsMKLDNN() is closer to OperatorWithKernel and needs **operation type** to proceed. 
I think both of them can be left separated for now, otherwise either CanMKLDNNBeUsed needs **op_type** to be passed or we start to base deciding on GetExpectedKernelType on OperatorWithKernel ::SupportsMKLDNN() as it has all the info needed and get rid of platform::CanMKLDNNBeUsed() as it is used mainly at the OperatorWithKernel level
What is your opinion on that ?